pkg/costmodel/cluster_helpers.go

```diff
@@ -191,12 +191,6 @@ func buildGPUCostMap(
 	gpuCostMap := make(map[NodeIdentifier]float64)
 	clusterAndNameToType := make(map[nodeIdentifierNoProviderID]string)
 
-	customPricingEnabled := provider.CustomPricesEnabled(cp)
```

Comment:
Big thanks to @ameijer for figuring out that custom pricing was overriding the provider's GPU cost settings.